webpack-entries.js

```diff
@@ -50,6 +50,7 @@ const entryPoints = {
     'dokan-category-commission': '/src/admin/components/Commission/index.js',
     'core-store': '/src/stores/core/store.ts',
     'dokan-status': '/src/Status/index.tsx',
+    'dokan-ui-style': '/node_modules/@getdokan/dokan-ui/dist/dokan-ui.css',
```

Comment:
- Could we avoid this?
- Will it server the purpose if we import in the base/tailwind css?